content/en/synthetics/browser_tests/test_runs.md

```diff
@@ -0,0 +1,190 @@
+
+**Monitor**
+: This section contains the name of the [Synthetic test's monitor][13] and the notification message.
+
+**CI/CD Execution**
+: This section contains a dropdown menu to change the [execution rule][12] for this test running in a CI/CD pipeline.
+
+## Test history
+
+In the **History** section, you can see three graphs:
+
+- The **Global Uptime** graph displays the total uptime of all test locations in a given time interval. 
+- The **Time-to-interactive by location and device** graph displays the amount of time in seconds the [last Long Task][15] takes to complete in a given time interval. For more information about uptime monitoring, see the [Website Uptime Monitoring with SLOs][14] guide.
+- The **Test duration by location and device** graph displays the amount of time in minutes each location and device takes to complete in a given time interval. 
+
+{{< img src="synthetics/browser_tests/history.png" alt="The History and Sample Runs section in the Test Details page" style="width=80%" >}}
+
+## Sample runs
+
+Browser test runs include components such as [screenshots](#screenshots), [page performance data](#page-performance), [errors](#errors), [resources](#resources), and [backend traces](#backend-traces) to help troubleshoot your [test failure](#failed-test-runs).
+
+In the **Sample Runs** section, you can examine the latest failed sample runs and compare them to recent successful sample runs.
+
+### Overview attributes
+
+Status
+: The status of your test run (`PASSED` or `FAILED`).
+
+Starting URL
+: The URL of your browser test scenario.
+
+Steps
+: The number of test steps completed in your sample run.
+
+Duration
+: The amount of time it took your test to run.
+
+Location
+: The managed or private location your test was executed from.
+
+Device
+: The type of device your test was executed from.
+
+Browser
+: The type of browser your test was executed from.
+
+Time ran
+: The amount of time that has passed since your test ran.
+
+Run type
+: The type of test run (CI, fast retry, manually triggered, or scheduled).
+
+### RUM sessions
+
+To view related sessions and available replays in the RUM Explorer, click **View Session in RUM**. To access a user session for a particular action or step in Session Replay, click **Replay Session**. For more information, see [Test Coverage][16].
+
+### Screenshots and actions
+
+Every executed test step contains a screenshot of the test action, a link to the session in Session Replay, the test action description, the starting URL, the step ID, the step duration, and page performance information.
+
+### Page performance
+
+Synthetic Monitoring displays two [Core Web Vital metrics][6] ([Largest Contentful Paint][2] and [Cumulative Layout Shift][3]) as pills to the right of each step URL. [First Input Delay][4] is available if you are using [Real User Monitoring][5] to collect real user data. For more information, see [Monitoring Page Performance][6].
```

Comment:
```suggestion
Synthetic Monitoring includes two [Core Web Vital metrics][6] ([Largest Contentful Paint][2] and [Cumulative Layout Shift][3]) as lab metrics, displayed as pills to the right of each step URL. [First Input Delay][4] is available if you are using [Real User Monitoring][5] to collect real user data. For more information, see [Monitoring Page Performance][6].
```

Should we also update the `Monitoring Page Performance` to reference lab metrics since it doesn't have that currently? https://docs-staging.datadoghq.com/alai97/browser-test-results-doc-update/real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals

Reply:
Sure, does this update make sense (https://github.com/DataDog/documentation/pull/16605) or is there additional content we can add?